nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -1422,6 +1424,11 @@ private void checkFieldInitialization(ClassTree tree, VisitorState state) {
       constructorInitInfo = checkConstructorInitialization(entities, state);
       notInitializedInConstructors = ImmutableSet.copyOf(constructorInitInfo.values());
     }
+    notInitializedInConstructors =
```

Comment:
Possibly worth a brief comment above this line:

```
// Filter out final fields, since javac will already check initialization
```

The idea is pretty obvious in retrospect, but it did take me a second to understand why were doing this without looking at the PR description 🙂

Reply:
Ok, I added that comment.